src/ucp/core/ucp_ep.h

```diff
@@ -159,9 +159,12 @@ enum {
     UCP_EP_INIT_CONNECT_TO_IFACE_ONLY  = UCS_BIT(7),  /**< Select transports which
                                                            support CONNECT_TO_IFACE
                                                            mode only */
-    UCP_EP_INIT_CREATE_AM_LANE_ONLY    = UCS_BIT(8),  /**< Endpoint requires an AM lane only */
-    UCP_EP_INIT_ALLOW_AUX_TL_RSC       = UCS_BIT(9)   /**< Endpoint allows selecting of auxiliary
-                                                           transports for its configuration */
+    UCP_EP_INIT_DISALLOW_AUX_TL_RSC    = UCS_BIT(8),  /**< Endpoint disallows selecting of auxiliary
```

Comment:
can we instead assume that if `UCP_EP_INIT_ALLOW_AUX_TL_RSC` is not present in the flag, than using aux is prohibited? (and update all places where it needs to be set)

Reply:
discussed offline, can still use 2 flags, but with different meaning:
 - flag allowing usage of aux transport for AM lane
 - flag allowing usage of aux transport for KA lane

Reply:
discussed offline